Elevel/Elevel.Api/Controllers/UserController.cs

```diff
@@ -25,5 +25,20 @@ public async Task<IActionResult> GetTokenAsync(TokenRequestModel model)
             var result = await _userService.GetTokenAsync(model);
             return Ok(result);
         }
+
+        [HttpGet("token")]
+        public async Task<IActionResult> UserInfo(string token, CancellationToken cancellationToken)
```

Comment:
and you should wrap this by Authorize token, and you should not have token in params